pkg/sessions/redis/redis_store.go

```diff
@@ -33,27 +33,27 @@ type TicketData struct {
 type SessionStore struct {
 	CookieCipher  *encryption.Cipher
 	CookieOptions *options.CookieOptions
-	Client        *redis.Client
+	Cmdable       redis.Cmdable
 }
 
 // NewRedisSessionStore initialises a new instance of the SessionStore from
 // the configuration given
 func NewRedisSessionStore(opts *options.SessionOptions, cookieOpts *options.CookieOptions) (sessions.SessionStore, error) {
-	client, err := newRedisClient(opts.RedisStoreOptions)
+	cmdable, err := newRedisClient(opts.RedisStoreOptions)
 	if err != nil {
 		return nil, fmt.Errorf("error constructing redis client: %v", err)
 	}
 
 	rs := &SessionStore{
-		Client:        client,
+		Cmdable:       cmdable,
 		CookieCipher:  opts.Cipher,
 		CookieOptions: cookieOpts,
 	}
 	return rs, nil
 
 }
 
-func newRedisClient(opts options.RedisStoreOptions) (*redis.Client, error) {
+func newRedisClient(opts options.RedisStoreOptions) (redis.Cmdable, error) {
```

Comment:
Perhaps rename this function given it now returns a different interface?

Reply:
renamed function name newRedisClient to newRedisCmdable